### ISSUE TITLE
Qestion: Mobile view

### ISSUE THREAD

**Issue by Fornori** [react-bootstrap-table/react-bootstrap-table2#819]
Hello.
How about support mobile view of the table?
From the box table not looks good. $( It's just adjust the column and that's all. Not good view on the mobile.
Is any suggest to adaptive this component for the mobile view?

**Comment by AllenFang** [comment]
@Fornori so far I dont have plan to do this, the feasible solution will be show a key value table instead in narrow device. [This](https://css-tricks.com/responsive-data-tables/) is sample.

**Comment by nathan-fortier** [comment]
I am actually working on this right now. I plan to tie into the column object and add additional parameters to choose which columns will be show on a mobile device. However I don't want to keep the formatting because a table just doesn't display data very well on mobile. Instead i'm thinking ill re-render the grid either with 1 column (if the columns support custom formatting as ill be honest i havn't done a lot of research yet) OR possibly just remove the grid altogether and replace it with another system. I don't want to have to reinvent the wheel in terms of pagination and filtering tho so this is definitely an area were if you have any input it would be good. 

I'd like it to scale down to a vertical list of tiles but also support the filtering and pagination features of the grid. Where each tile can use a property or two from the column object to determine if it should be shown and if its a title or a detail.

**Comment by Fornori** [comment]
Yep, for now, I use react-mobile-detect and if mobile change the grid with the card....

**Comment by nathan-fortier** [comment]
@Fornori what are you using for your card? Self built or another node module?

**Comment by Fornori** [comment]
> @Fornori what are you using for your card? Self built or another node module?

I use Card from "reactstrap"

**Comment by LamarrD** [comment]
@nathan-fortier Any update on this, if not I can take a shot at implementing this sometime soon
